src/web/mjs/connectors/BilibiliComics.mjs

```diff
@@ -13,7 +13,7 @@ export default class BilibiliComics extends BilibiliManhua {
         };
         //override BilibiliManhua config as we need lanuage switch. Difference is handled in the main code with (this.config.language) ? : test
         this.config = {
```

Comment:
Can this be simplified by modifying the `config` field instead of redefining everything again.
Consider to use `delete` or assign `undefined` to object properties that shall be removed from `super.config` and `super.config['key'] = { /* ... */ }` to add new properties.

Reply:
i used the method you advised last time. Indeed, your new idea seems to be better

Reply:
Weird : This one works flawlessly 

```
        this.config.language.options = [
            { value: 'fr', name: 'French' },
            { value: 'en', name: 'English' },
            { value: 'cn', name: 'Chinese' },
            { value: 'id', name: 'Indonesian' },
            { value: 'es', name: 'Spanish' },
        ];
```

but this one tells me 'picquality' is undefined
```
        this.config.picquality.options =
        [
            { value: 'raw', name: 'Raw' },
            { value: 'hd', name: 'HD' },
            { value: 'sd', name: 'SD' },
        ];
```

when it clearly exist in parent class